Source/DiabloUI/art.cpp

```diff
@@ -102,7 +102,7 @@ Uint32 GetPcxSdlPixelFormat(unsigned bpp)
 
 } // namespace
 
-void LoadArt(const char *pszFile, Art *art, int frames, SDL_Color *pPalette, const std::array<uint8_t, 256> *colorMapping)
+void LoadArt(const char *pszFile, Art *art, int frames, SDL_Color *pPalette, const std::array<uint8_t, 256> *colorMapping, bool showError)

```

Comment:
We only have one call site that passes `showError` = `false`.
Could we remove this callsite (see other comment) or should we add a `ExistsAsset` method instead of adding this to `LoadArt`?

Reply:
up to you : P

Reply:
If no one uses it => remove it, cause less code is better code. 😜 
But I'm [not sure](https://i2.wp.com/scifiinterfaces.com/wp-content/uploads/2018/10/Idiocracy_ippa05.png?resize=768%2C432&ssl=1) if no one uses it.

Reply:
@AJenbo I looked in the git history and it seems like you added the logic in `LoadHeros`. What do you think?

Reply:
I would prefer to keep it as I am working on content for it to load :sweat_smile:

Reply:
I think we should introduce something like a `ExistsAsset` method and crash if any error happens in `LoadArt` that didn't create a surface.
We do the same thing for sounds (see `sound_file_load`) and I think that helps to diagnose odd setups easier.